givitsite/coordinate/models.py

```diff
@@ -1,3 +1,42 @@
 from django.db import models
+from django import forms
+from friendreq.models import REGION_CHOICES, ITEM_CHOICES , ItemRequest 
 
-# Create your models here.
+
+
+class CoordinatedItems(models.Model):    
+    item = models.CharField(max_length = 40,choices = ITEM_CHOICES)
+    request_id = models.IntegerField(default = 100)
+    date = models.DateField(auto_now=False, auto_now_add=False)
+    student_phone_number = models.CharField(max_length = 10)
+    pickup_time = models.TimeField(auto_now=False, auto_now_add=False)
+    pickup_location = models.CharField(max_length = 40,choices = REGION_CHOICES)
+    drop_off_time = models.TimeField(auto_now=False, auto_now_add=False)
+    drop_off_location = models.CharField(max_length = 40,choices = REGION_CHOICES)
+
+
+
+class DateInput(forms.DateInput):
+    input_type = 'date'
```

Comment:
I assume you're doing this (and TimeInput) so you can use a DatePicker/TimePicker on the form?

Reply:
yes that's the goal:)